### DISCUSSION TITLE
站点在配置了伪静态的情况下使用http方式申请SSL证书失败

### DISCUSSION THREAD

**Discussion by zhengjianyang** [1Panel-dev/1Panel#6068] [Q&A]
### 1Panel 版本

v1.10.11-lts

### 问题描述

比如我配置了一段nginx伪静态规则：
location / {
    if (!-e $request_filename) {
        rewrite ^(.*)$ /index.php?s=/$1 last;
        break;
    }
}

再申请ssl证书就会访问不了验证文件，因为我的规则是命中所有非文件请求，验证文件又不存在于站点目录下，就无法请求到验证文件了。

### 重现步骤

location / {
    if (!-e $request_filename) {
        rewrite ^(.*)$ /index.php?s=/$1 last;
        break;
    }
}

加一段nginx伪静态规则申请SSL证书

### 期待的正确结果

在有一些特别伪静态规则情况下还能正常文件到SSL申请验证文件即正常申请SSL证书

**Comment by zhengkunwang223** [comment]
您好 经过我们测试 添加此伪静态配置之后 使用 HTTP 模式申请网站不受影响
麻烦提供更多日志

**Comment by rioufbi** [comment]
在伪静态里第一条就写
rewrite ^/.well-known/acme-challenge/(.*)$ /.well-known/acme-challenge/$1 break;
如果有其他申请SSL的路径就自己修改一下。
我写了之后就正常可以获取到了，感觉像是BUG。
